### ISSUE TITLE
Summary output from highstate should include target id

### ISSUE THREAD

**Issue by Bilge** [saltstack/salt#20468]
Typical output from `state.highstate` currently appears as the following:

>Summary
\-------------
Succeeded: 59 (changed=52)
Failed:     0
\-------------
Total states run:     59

This is terrible, particularly when targeting multiple machines, since one has to scroll past 59 states to find which machine the summary pertains to. At the very least it should include the target id in the summary in addition to the head.

Moreover, it would be better to see a summary for all machines together instead of one summary for each machine.

**Comment by jfindlay** [comment]
@Bilge, great idea.  There are many improvements that could be made to the CLI outputter.

**Comment by richardhansen** [comment]
I believe this was fixed by PR #24242

**Comment by rallytime** [comment]
@richardhansen Good find - that looks correct to me. I just tried a highstate of 2015.8, and it includes the minion id in the summary.

With `salt-call --local state.highstate`:
```
Summary for local
------------
Succeeded: 1
Failed:    0
------------
Total states run:     1
Total run time:   7.237 s
```
And with `salt rallytime state.highstate`:
```
Summary for rallytime
------------
Succeeded: 1
Failed:    0
------------
Total states run:     1
Total run time:   7.347 s
```
@Bilge I am going to go ahead and close this, but the functionality you're requesting here is available in 2015.8.0. If this pops up again let us know and we will be glad to re-open. Thanks!
